Parse-Dashboard/tests/.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Not needed, already exists under `src/lib/tests/.eslintrc.json`

Reply:
I deleted it.